### PR TITLE
Typos in browser.dtd

### PR DESCRIPTION
**Pull request by piotrdrag**
I've found some minor typos. I'm not sure why this pull request shows all the commits I ever made to this repo, but just one file changed (from the last commit). Let's hope it works.

### CODE REVIEW

**Comment by splewako** [comment]
> I'm not sure why this pull request shows all the commits I ever made to this repo

You are committing on the `branches/default/tip` branch and even merged with remote but remote still has (and will) different history.

I would suggest using separate topic branches for pull requests and leaving `branches/default/tip` as upstream tracker and branch starting point.

File: browser/chrome/browser/browser.dtd [hunk 1]
```diff
@@ -118,9 +118,9 @@ These should match what Safari and other Apple applications use on OS X Lion. --
 
 <!ENTITY fullscreenWarning.beforeDomain.label "">
 <!ENTITY fullscreenWarning.afterDomain.label "działa teraz w trybie pełnoekranowym">
-<!ENTITY fullscreenWarning.generic.label "Dokument jest teraz wyśitelany w trybie pełnoekranowym">
+<!ENTITY fullscreenWarning.generic.label "Dokument jest teraz wyświetlany w trybie pełnoekranowym">
 <!ENTITY exitDOMFullscreen.button "Opuść tryb pełnoekranowy (Esc)">
-<!ENTITY exitDOMFullscreenMac.button "Opuść tryb pełnoekranowy (esc)">
+<!ENTITY exitDOMFullscreenMac.button "Opuść tryb pełnoekranowy (Esc)">
```
Comment (splewako) [inline browser/chrome/browser/browser.dtd:123]: Lowercase "esc" is [intentional](https://github.com/mozilla/gecko-dev/blame/master/browser/locales/en-US/chrome/browser/browser.dtd#L143) as it is labeled that way on mac keyboards.

File: browser/chrome/browser/browser.dtd [hunk 2]
```diff
@@ -743,7 +743,7 @@ you can use these alternative items. Otherwise, their values should be empty. --
 <!ENTITY trackingContentBlocked.block.accesskey "W">
 <!ENTITY trackingContentBlocked.disabled.message "Ochrona przed śledzeniem jest wyłączona">
 
-<!ENTITY pointerLock.notification.message "Naciśnij ESC w dowolnej chwili, aby wyświetlić go ponownie.">
+<!ENTITY pointerLock.notification.message "Naciśnij Esc w dowolnej chwili, aby wyświetlić go ponownie.">
```
Comment (splewako) [inline browser/chrome/browser/browser.dtd:746]: In [en-US](https://github.com/mozilla/gecko-dev/blame/master/browser/locales/en-US/chrome/browser/browser.dtd#L834) we have "ESC" here but I didn't found a reason, so "Esc" should be fine.

**Comment by piotrdrag** [comment]
I will read up on branching and get back to you.

**Comment by splewako** [comment]
See also:
- https://guides.github.com/introduction/flow/
- http://scottchacon.com/2011/08/31/github-flow.html

**Comment by piotrdrag** [comment]
Thanks!

**Pull request closed without merging**